tensorboard/webapp/notification_center/_views/notification_center_test.ts

```diff
@@ -48,6 +49,7 @@ describe('notification center', () => {
           initialState: [],
         }),
       ],
+      schemas: [NO_ERRORS_SCHEMA],
```

Comment:
why `NO_ERRORS_SCHEMA` is needed?

Reply:
Schema and NO_ERROR_SCHEMA specifically allows us to omit certain Angular components defined in the template. 

To be more specific, we do not absolutely need the real implementation of the `markdown-renderer` to test the `notification-center` so the real implementation is omitted.